pages/documentation/developer/scripting/python.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+---
+
+# Python scripting
+
+The [pypowsybl](../repositories/pypowsybl.md) provides a way to use all the features of PowSyBl in python script. This Python integration relies on the [JPype](https://www.py4j.org) project that provides full access to Java from Python. This is achieved not through re-implementing Python, as [Jython](https://www.jython.org/) has done, but rather through interfacing at the native level in both virtual machines.
+
+This shared memory based approach achieves decent computing performance, while providing the access to the entirety of CPython and Java libraries. For optimal performance, we recommend to use [Groovy scripts](groovy.md) or implement part of your script directly in Java.
```

Comment:
Fix: entirely

Reply:
[Entirety](https://www.wordreference.com/synonyms/entirety) seems OK in that context.